app/setup.py

```diff
@@ -228,36 +227,26 @@ def after_request(response):  # pylint: disable=unused-variable
     return application
 
 
-def _get_cdn_root_url(application) -> str:
-    parsed_url = urlparse(application.config["CDN_URL"])
-    return f"{parsed_url.scheme}://{parsed_url.hostname}"
-
-
-def add_cdn_url_to_csp_policies(application) -> Dict:
+def add_cdn_url_to_csp_policies(cdn_url) -> Dict:
```

Comment:
I've made some suggestions  to fix naming inconsistencies around csp policy rather than policies
```suggestion
def add_cdn_url_to_csp_policy(cdn_url) -> Dict:
```